src/components/chrome/chrome.module.scss

```diff
@@ -1,28 +1,63 @@
-@import '~@elastic/eui/src/global_styling/functions/_index.scss';
-@import '~@elastic/eui/src/global_styling/variables/_index.scss';
```

Comment:
Oh wow the work you've done on this! 💯 

Out of curiosity, is there a reason the importing of the EUI globals was removed here? I'm seeing a lot of new hard-coded hex values for colors now instead of just using the SASS tokens. Keeping them imported would give consumers a good place to start with consuming the SASS tokens.

Reply:
Ah 😅 

The hex values are there because I was copying-and-pasting values from DevTools to mirror the style guide. I actually took the imports out because it was causing CSS hot reloads to be super-slow. I can look into it more if you like.

Reply:
It's probably worth trying to demonstrate better patterns, actually, so I'll have another look.